scripts/run_presubmit_checks.sh

```diff
@@ -41,6 +42,12 @@ then
 fi
 
 set -e
+# Run third party folder size check.
+echo 'Running third party size check'
```

Comment:
Instead of calling bash, you should be able to just call the python script directly (see the linter), it is safe to assume that the developer would have run `bash scripts/start.sh`, since that's the standard for some time now.

Reply:
Done (https://github.com/oppia/oppia/pull/5743/commits/f558a9004cf45bac8835ab9f95035633b5ce2820)